### ISSUE TITLE
Global | Manage focus on review & submit page on submit error

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#94687]
## What happened?

**Describe what went wrong**:

When you click on submit button on the review and submit page, focus is lost.

**Add the steps to reproduce the bug**:

1. Go to `https://staging.va.gov/decision-reviews/test-sc/` and start filling out the form.
2. Navigate to `https://staging.va.gov/decision-reviews/test-sc/review-and-submit`
3. Submit the form without checking the privacy policy checkbox
4. Note that focus is lost.
5. Open the "Issues for review" accordion
6. Edit the first page
7. Uncheck all issues
8. Scroll down and submit the form again (skip the "Update page" button)
9. Note that focus is lost

**Add a screen recording or photo**:

![Image](https://github.com/user-attachments/assets/2ba6243e-7860-429a-a524-3a958feeb0dd)

## How severe is the problem?

- [x] **Critical**: User is block or significantly hindered from submitting the form, data is lost, or claim and evidence test coverage is missing. Fix ASAP.
- [ ] **Moderate**: User is not blocked, but the happy path is interrupted. Fix after critical bugs.
- [ ] **Minor**: Cosmetic bugs and minor language inconsistencies. Defer to a later sprint.

## What is the desired behavior?

1. _When a user activates the submit button, the is focus is shifted._

We should shift focus to the `checkbox` field of the privacy policy, and if checked and an error alert becomes visible, focus on the alert

## Definition of Done

- [ ] Acceptance criteria reviewed and approved by product or design
- [ ] Unit test (or other test coverage) added if possible
- [ ] Logging and monitoring in place

**Comment by comaurice** [comment]
Pending a code review, CI tests are failing right now

**Comment by kayline** [comment]
Waiting to merge until after prod deploy today

**Comment by Mottie** [comment]
@davidakennedy Please review these changes in staging

**Comment by davidakennedy** [comment]
@Mottie Gave this another test, and both areas work as expected. On the privacy checkbox, focus moves to the error message text. On the no issues selected, focus moves to the error container.
